tutorials/haskell/hspec-golden-test-tutorial/code/test/Json/JsonGoldenSpec.hs

```diff
@@ -0,0 +1,30 @@
+module Json.JsonGoldenSpec where
+
+import           Test.Hspec
+import           Test.Hspec.Golden
+import           JSON.Json
+import qualified Data.ByteString.Lazy as B         
+
+
+goldenBytestring :: String -> B.ByteString -> Golden B.ByteString
+goldenBytestring name actualOutput =
+    Golden {
+        output = actualOutput,
+        encodePretty = show,
+        writeToFile = B.writeFile,
+        readFromFile = B.readFile,
+        testName = name,
+        directory = ".otherGolden"
+          
+    }
+
+
+spec :: Spec
+spec =
+
+  describe "encodeCountries" $ do
+--   it "encodes a group of Countries into a JSON String " $
+--    defaultGolden "json" (encodeCountries countries)
```

Comment:
@Jagl257 Probably, we don't need this comment.

Reply:
Actually I left that one one purpose cause in the tutorial I explain that using the default golden function will fail. I left that for the people to find that while reading the tutorial. What do you think ?

Reply:
Mmm in that, could you please add a note above that? Also, I think a block can be more useful:
```hs
{-
    NOTE: `defaultGolden` doesn't work for outputs different that String
    it "encodes a group of Countries into a JSON String " $
      defaultGolden "json" (encodeCountries countries)
-}
```

Reply:
Yes it looks better that way @CristhianMotoche, alredy done.